.travis.yml

```diff
@@ -31,11 +33,6 @@ install:
 - python bootstrap.py
 - bin/buildout $VERSIONS annotate
 - bin/buildout $VERSIONS
-before_script:
-  - export ROBOT_SELENIUM2LIBRARY_RUN_ON_FAILURE="Capture Page Screenshot"
```

Comment:
don't remove this line.

Reply:
@hvelarde ok, remove all of the 'before_script' part because it wasn't in the comment of @idgserpro